### DISCUSSION TITLE
代码里有config.json又有auth.json，api_key.txt，应该是怎么配置的？

### DISCUSSION THREAD

**Discussion by cgofun** [GaiZhenbiao/ChuanhuChatGPT#508] [General]
我用了最新版，配置了config.json，非docker模式，总是提示key错误，我看代码里有config.json又有auth.json，api_key.txt，应该是怎么配置的？

**Comment by Keldos-Li** [comment]
是我们没写清楚 对不起😭 为保证之前用txt配置的向前兼容性，txt中的配置会覆盖json的配置。（应该是）

**Comment by cgofun** [comment]
感谢你们提供这么好的作品，我也是发现问题，发来探讨一下
![image](https://user-images.githubusercontent.com/7732886/229332247-91553039-6487-4fc6-ace5-1c4a03f8432a.png)
之前的版本都是正常的，这个版本我搞不定了。我现在只用了config.json，我想之后会统一用这个吧，但总是提示key错误，页面上输入key也是同样错误，就是上面截图这个提示。

**Reply by Keldos-Li** [reply]
这个是因为openai把获取账单的接口干掉了，参考 #505 不过对话功能应该没问题

**Comment by cgofun** [comment]
明白了，期待修正。
另外有几句代码，建议加入
![image](https://user-images.githubusercontent.com/7732886/229334826-50be1380-3e4c-4ba5-8c78-b30a5733e883.png)

**Comment by Keldos-Li** [comment]
现在默认为config.json，旧的配置会进行转换。
